src/CMakeLists.txt

```diff
@@ -22,8 +24,10 @@ source_group("Sources" FILES ${sources})
 
 add_executable(aqw-opcua-server ${headers} ${sources})
 
+find_package(cpprestsdk CONFIG REQUIRED)
```

Comment:
My understanding is that it is better to have this `find_package` in main `CMakeLists.txt` so that this dependency is clearly visible from just looking at main configuration file.

Reply:
Think I moved it because of some cmake GUI errors. Probably because I was putting in wrong source folder or didn't clear build folder/cache. Works in main file now - moved back.